website/docs/utilities/dialog-primitive/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,23 @@
+## Usage
+
+The DialogPrimitive sub-components are not intended to be used independently of each other or in isolation. If you are looking to create your own dialog-based component, use the `DialogPrimitive::Wrapper` as your foundation.
+
+A common example of this is a non-modal dialog such as a SplitWindow or Panel which allows the user to interact with both the main page content and the content within the dialog.
+
+!!! Info
+
+If you discover a use case outside of this example, please [contact](/about/support) the HDS team for assistance.
+!!!
+
+![Example of a non-modal example](/assets/components/dialog-primitives/dialog-primitive-non-modal-example.png)
+
+### Use within HDS components
+
+Examples of how to use the DialogPrimitive components can be found in the HDS [Modal](/components/modal) and [Flyout](/components/flyout) components which are composed using the DialogPrimitive. They are both types of dialogs, but have different use cases depending on the context and type of information they convey.
+
+### What is a “dialog”
```

Comment:
(suggestion, non-blocking)

If we can't add a ? to the end of "What is a dialog"

Maybe consider changing the title to "Defining dialog" or something that is a statement instead of a question.

Reply:
(looking at the history of the documentation this content is coming from, I think it's you who wrote this: https://github.com/hashicorp/design-system/blame/51fb2709729291cf6932fab5e90e947b11592062/website/docs/utilities/dialog-primitive/partials/guidelines/guidelines.md)

Reply:
Personally I like "What is a dialog"

Reply:
I agree with either adding a question mark or rewriting the title

Reply:
Added a question mark.